src/operations/create/core.rs

```diff
@@ -1,90 +1,152 @@
 //! Defines the core functions for creating records. These functions should not be called directly
 //! from the Python API but rather from the TCP connection in the runtime module. In this
 //! module we can do the following:
-//! 
+//!
 //! * Create a record in the database
+use crate::connection::interface::WrappedConnection;
 use serde_json::value::Value;
 use surrealdb::opt::Resource;
-use surrealdb::sql::Range;
-use crate::connection::interface::WrappedConnection;
-
+use surrealdb::sql::Thing;
```

Comment:
This is no longer a documented API in v2. Consider using `surrealdb::RecordId` instead.

Reply:
it should be `surrealdb/surrealdb:v2.0.2`

Reply:
This implementation requires accessing fields on Thing, so we have to be exposed to this api